tests/test_integrated_channels/test_integrated_channel/test_exporters/test_learner_data.py

```diff
@@ -252,6 +252,7 @@ def test_learner_data_instructor_paced_no_certificate(
         assert learner_data[1].course_id == self.course_id
 
         for report in learner_data:
+            assert report.user_email == 'burneremail@aol.com'
```

Comment:
nit: rather than `burneremail@aol.com` should we use something like `self.user_2.email` ?